crates/uv-resolver/src/resolver/mod.rs

```diff
@@ -342,6 +344,9 @@ impl<
                                     UnavailablePackage::NotFound => {
                                         "was not found in the package registry"
                                     }
+                                    UnavailablePackage::Unparseable => {
+                                        "was found, but the metadata could not be parsed"
+                                    }
```

Comment:
👍 This is the kind of thing that would be improved by generic metadata in PubGrub... I started work on it but it's a lot.